### DISCUSSION TITLE
My achievements is not showing on my profile

### DISCUSSION THREAD

**Discussion by Sahad-09** [community/community#42219] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Even after enabling my achievements through settings it still doesn't show on my profile.
Please help

**Comment by radar-blip** [comment]
If you have any achievements, they should show up here: https://github.com/Sahad-09?tab=achievements
If they are there, just un-hide the ones you want to display. 👍🏻

**Comment by Luis-Angel-Santos** [comment]
Your achievements can be viewed and configured here: https://github.com/(yourusername)?tab=achievements

- You can learn more about achievements on the github blog; [GitHub Blog](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/)

- If you want to get your first achievement, I recommend you to check this : https://github.com/community/community/discussions/18294
